service/client.go

```diff
@@ -23,6 +24,7 @@ func (c Client) GetJobs(parentID string) ([]database.Job, error) {
 		c.Logger.Error("Error loading jobs from DB", parentID)
 		return nil, err
 	}
+	sort.Sort(database.ByCreatedAt(jobs))
```

Comment:
You could also use ``sort.Slice`` so you don't need to implement ``sort.Interface``:

```
sort.Slice(jobs, func(i, j int) bool { return return jobs[i].CreatedAt.Before(jobs[j].CreatedAt) })
```